pages/common/gocryptfs.md

```diff
@@ -0,0 +1,23 @@
+# gocryptfs
+
+> Encrypted overlay filesystem written in Go.
+
+- Initialize encrypted filesystem:
+
+`gocryptfs -init {{path/to/cipher_dir}}`
+
+- Mount encrypted filesystem:
+
+`gocryptfs {{path/to/cipher_dir}} {{path/to/mount_point}}`
+
+- Mount with explicit master key instead of password:
+
+`gocryptfs --masterkey {{path/to/cipher_dir}} {{path/to/mount_point}}`
+
+- Change password:
+
+`gocryptfs --passwd {{path/to/cipher_dir}}`
+
+- Make encrypted view of a plain-text directory:
```

Comment:
I've use *cipher* because the official documentation use *cipher*. I think is better use the same vocabulary of official documentation.

---
> Is it just an encrypted directory ? Then we should just call that.

Not exactly, it is an encrypted pic of a plain dir in a determinate time. I've change *view* with *snapshot*. I think that this is clearer